searchlib/src/vespa/searchlib/engine/trace.h

```diff
@@ -76,17 +82,20 @@ class Trace
     void done();
 
     vespalib::string toString() const;
-    Cursor & getRoot() const { return _root; }
-    vespalib::Slime & getSlime() const { return *_trace; }
+    Cursor * getRoot() const { return _root; }
```

Comment:
should call lazy construct, could add has_root predicate to avoid creating if not needed.

Reply:
Fixed as discussed